CMakeLists.txt

```diff
@@ -1,13 +1,22 @@
 CMAKE_MINIMUM_REQUIRED(VERSION 3.1)
 
-project(BATCHEDMAT LANGUAGES CXX)
+project(NEML2 LANGUAGES CXX)
 
 # ## Setup modules ###
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/Modules/")
 
 # Werror, etc
 add_compile_options(-Wall -Wextra -pedantic -Werror)
 
+# This is aggravating but occurs because pre-packaged torch uses
+# _GLIBCXX_USE_CXX11_ABI=0.  If you roll your own torch this may
+# not occur, so this flag could be dangerous in the future
+option(CXX11ABI "Whether to let GLIBCXX use cxx11 ABI" OFF)
+
+if(NOT CXX11ABI)
+      add_compile_definitions(_GLIBCXX_USE_CXX11_ABI=0)
+endif()
+
```

Comment:
You may want to check this. I am not sure if this is the best way to go, i.e., should we let the user control this parameter, or is there a way to figure out whether libtorch is built with or without CXX11 ABI? Also, do we default this to ON or OFF?